metadata/demographic/metro_micro_statistical_areas_schema.md

```diff
@@ -0,0 +1,57 @@
+# Title
+
+Metro Micro Statistical Areas
+
+## ID
+
+4e8eaf1f-eda3-47bb-974f-f99be23847b3
+
+### namelsad
+
+Current name and the translated legal/statistical description for metropolitan statistical area/micropolitan statistical area
+
+### lsad
+
+Current legal/statistical area description code for metropolitan statistical area/micropolitan statistical area.
+
+Values:
+
+- M1 = Metropolitan Statistical Area
+- M2 = Micropolitan Statistical Area
+
+### memi
+
+Current metropolitan/micropolitan status indicator.
+
+Values:
+
+- 1 = Metropolitan
+- 2 = Micropolitan
+
+### mtfcc
+
+MAF/TIGER feature class code.
+
+### funcstat
+
+<!--- not sure what this field is referring to. --->
+
+### aland
+
+Current land area in square meters.
+
+### awater
+
+Current water area in square meters.
+
+### intptlat
+
+Current latitude of the internal point.
+
+### intptlon
+
+Current longitude of the internal point
+
+### partflg
```

Comment:
I think this is supposed to be a flag indicating if the area is part of a combined statistical area, but can't find explicit confirmation.